modules/listing-alerts-that-are-firing.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="listing-alerts-that-are-firing_{context}"]
+= Listing alerts that are firing
+
+Alerts provide notifications that a set of circumstances are apparent within an {product-title} cluster. You can review the alerts that are firing in your cluster by using the Alerting UI in the {product-title} web console.
```

Comment:
Or,

> Alerts provide notifications about exceptional events within an {product-title} cluster.

Reply:
My view is that the events might not be exceptional. Some alerts are informational. Alerts fire when explicit conditions defined in alerting rules are true for a defined time period. I have updated this sentence to the following:

"Alerts provide notifications that a set of defined conditions are true in an {product-title} cluster."

Reply:
There is also an 'Additional resources' link following this module to the [Managing alerts](https://docs.openshift.com/container-platform/4.6/monitoring/managing-alerts.html) section so that readers can learn more about alerting.